css/styles.css

```diff
@@ -993,3 +993,76 @@ nav .btn {
 		margin-right: 0;
 	}
 }
+
+.customer__cards {
+      display: flex;
+      height: 30rem;
+      max-width: 92rem;
+      
+    }
+
+    .customer_cards_2{
```

Comment:
Can we fix the formatting and indentation in this file -- there's extra indents and some random +/- 1 spaces between lines.